native_words.asm

```diff
@@ -3082,27 +3095,19 @@ do_common:
                 ; compile the address we need to LDA at runtime
                 dex
                 dex
-                lda cp
-                sta 0,x                 ; LSB
                 lda cp+1
                 sta 1,x                 ; MSB   ( limit start here )
+                lda cp
+                sta 0,x                 ; LSB
```

Comment:
swap so I can reuse A, realized we don't actually need to write anything, just advance cp